lib/pbench/cli/agent/commands/pbench_create_user.py

```diff
@@ -0,0 +1,101 @@
+import click
+import requests
+
+from pbench.agent.base import BaseCommand
+from pbench.cli.agent import pass_cli_context
+from pbench.cli.agent.options import common_options
+
+
+class PbenchCreateUser(BaseCommand):
+    def __init__(self, context):
+        super().__init__(context)
+
+    def execute(self):
+        """Creates a user in the pbench user database"""
+
+        server = self.config.agent.parser.get("results", "server_rest_url")
```

Comment:
This is interesting; I don't think we want this to be set up as an "agent" utility, with the `pbench-agent.cfg` setting for `server_rest_url`, which requires a `pbench-agent` installation. I think we want to base this on the `pbench-server.cfg` configuration and `rest_uri`. In theory we could allow users to self-register through the CLI, and conceivably that could make this also useful as part of the agent installation, but it's mostly a server administration tool. (And, selfishly, this does me no good for solving my server-side gold unittest problem if it depends on an agent configuration environment!)

It might not be a bad idea to try to normalize the server configuration settings between the agent and server side config files and possibly even move this to a `common` tool set that's not restricted to either server or agent. For simplicity, though, I suggest that we start out as a server management utility using server configuration.